### PR TITLE
Update PushNotifications-spec.md to match design changes.

### PR DESCRIPTION
**Pull request by pmpurifoy**
- Removed PushNotificationActivationInfo from idl
- Removed PushNotificationRegistrationActivators from idl
- Modified PushNotificationManager registration APIs
- Updated sample code
- Updated guidance on foreground handler registration

### CODE REVIEW

File: specs/PushNotifications/PushNotifications-spec.md [hunk 1]
```diff
@@ -80,10 +80,13 @@ Link to the official Windows App SDK timeline can be found
 ## In this scenario, the process that Registers the Push Activator and the process specified as the COM server are the same
 
 The code in Main would follow the pattern below:
-
--   The Registration will take in a CLSID of the COM server as part of the Activator setup. The
-    registration API will simply abstract away the COM details from the developer including
-    registration of the in-process Windows App SDK component.
+-   Both packaged and unpackaged apps will call into the registration API.
```
Comment (danielayala94) [inline specs/PushNotifications/PushNotifications-spec.md:83]: Nit: 
```suggestion
-   All apps will call into the registration API. 
```

File: specs/PushNotifications/PushNotifications-spec.md [hunk 2]
```diff
@@ -95,48 +98,48 @@ The code in Main would follow the pattern below:
...
+
+        // Do stuff to process the raw payload
+        std::string payloadString(payload.begin(), payload.end());
+        std::cout << "Push notification content received from FOREGROUND: " << payloadString << std::endl << std::endl;
+        args.Handled(true);
+    });
 
     // Registers the application to receive push notifications
-    PushNotificationManager::RegisterActivator(info);
+    PushNotificationManager::Default().Register();
```
Comment (danielayala94) [inline specs/PushNotifications/PushNotifications-spec.md:129]: Can we declare a local for PushNotificationManager::Default() and re-use it?

**Review by danielayala94 — APPROVED** [review]

**Comment by pmpurifoy** [comment]
Close this and moving to a new branch.

**Comment by pmpurifoy** [comment]
https://github.com/microsoft/WindowsAppSDK/pull/2196

**Pull request closed without merging**